R/tests.R

```diff
@@ -10,23 +10,18 @@ renv_tests_scope <- function(packages = character(), project = NULL, envir = par
 
   renv_tests_init()
   renv_tests_scope_repos(envir = envir)
+  renv_scope_envvars(R_LIBS_USER = NULL, envir = envir)
```

Comment:
`init()` changes this so this ensures we reset at the end of the test.

Reply:
Maybe worth adding as a comment in the code so we remember the reason?